spec/features/course_creation_spec.rb

```diff
@@ -346,6 +348,36 @@ def go_through_researchwrite_wizard
     end
   end
 
+  describe 'course created from a campaign', js: true do
+    let(:campaign) do
+      create(:campaign,
+             id: 10001,
+             title: 'My Awesome Campaign',
+             description: 'This is the best campaign')
+    end
+
+    before do
+      ENV['default_course_type'] = 'BasicCourse'
```

Comment:
instead of setting ENV and unsetting it, you can do `allow(Features).to receive(:default_course_type)`